### PR TITLE
 Accountancy - Add date_export on new ledger & subledger to show edit or delete link when is not export yet

### PR DESCRIPTION
**Pull request by aspangaro**
+ Repare link to create unknown account
+ Add language file

### CODE REVIEW

File: htdocs/accountancy/bookkeeping/balance.php [hunk 1]
```diff
@@ -321,8 +321,8 @@
 		$root_account_description = $tmparrayforrootaccount['label'];
 		$root_account_number = $tmparrayforrootaccount['account_number'];
 
-		if (empty($accountingaccountstatic->label) && $accountingaccountstatic->id > 0) {
-			$link = '<a href="'.DOL_URL_ROOT.'/accountancy/admin/card.php?action=update&token='.newToken().'&id='.$accountingaccountstatic->id.'">'.img_edit().'</a>';
+		if (empty($tmparrayforrootaccount['label'])) {
```
Comment (eldy) [inline htdocs/accountancy/bookkeeping/balance.php:324]: This bugs.
When you are here the accounting account already exists. Label may or may not exists.
If label does not exists and accountingaccountstatic already exists, we must link to update the account no to create.

**Comment by aspangaro** [comment]
this functionality is not present to update an accounting account (why not) but to identify accounting accounts that do not exist in the chart of accounts probably present as a result of an accounting data import and to help with the creation.

Can we mix the two systems? Update when the account is known and creation when the account is not known.

image

**Comment by eldy** [comment]
Current code manage the "update when label is not defined".
Yours is to add when no accounting exists.
So yes, best should be to manage both situation.

**Comment by aspangaro** [comment]
Before (v12), we manage create mode & in v13, we manage edit mode.

Now, when we manage the both situation : 
(For this image, I have removed the label of an existing account)

To my astonishment, I also realized that one could create an account in the chart of accounts without a number & without a label... I added a control on this point

image

**Pull request merged**